source/ns_cmdline.c

```diff
@@ -263,7 +272,9 @@ void cmd_init(cmd_print_t *outf)
         ns_list_init(&cmd.cmd_buffer);
         cmd.init = true;
     }
+#ifndef MBED_CLIENT_CLI_TRACE_ENABLE
     mbed_trace_exclude_filters_set(TRACE_GROUP);
+#endif
```

Comment:
Remove these three lines please. The purpose of the `mbed_trace_exclude_filters_set()` call here is just to disable traces by default. Since the new macros do that, this isn't needed.

It'll also work with mbed-trace too.